From c2e843f9386bfa01bfbdd2742ce89f6c474ac0ee Mon Sep 17 00:00:00 2001
From: Phil Sutter <psutter@redhat.com>
Date: Wed, 20 Jun 2018 09:23:54 +0200
Subject: [PATCH] nft.8: Fix reject statement documentation

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1571938
Upstream Status: nftables commit 9b3036bb9f00d

commit 9b3036bb9f00d6e244ed92e0e782c5617ae40b84
Author: Phil Sutter <phil@nwl.cc>
Date:   Wed Jun 6 10:56:26 2018 +0200

    nft.8: Fix reject statement documentation

    First of all, 'with icmp6' is invalid, expected is 'with icmpv6'. In
    addition to that, parameter 'type' expects an icmp*_code type, not
    icmp*_type. The respective table column was already correct, but in
    synopsis it was wrong.

    Signed-off-by: Phil Sutter <phil@nwl.cc>
    Signed-off-by: Florian Westphal <fw@strlen.de>
---
 doc/nft.xml | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/doc/nft.xml b/doc/nft.xml
index c7e12c9..3da5fda 100644
--- a/doc/nft.xml
+++ b/doc/nft.xml
@@ -4472,14 +4472,14 @@ ip6 filter output log flags all
 						<arg choice="none">with</arg>
 						<group choice="req">
 							<arg>icmp</arg>
-							<arg>icmp6</arg>
+							<arg>icmpv6</arg>
 							<arg>icmpx</arg>
 						</group>
 						<arg choice="none">type</arg>
 						<group choice="req">
-							<arg>icmp_type</arg>
-							<arg>icmp6_type</arg>
-							<arg>icmpx_type</arg>
+							<arg>icmp_code</arg>
+							<arg>icmpv6_code</arg>
+							<arg>icmpx_code</arg>
 						</group>
 					</arg>
 				</cmdsynopsis>
@@ -4516,7 +4516,7 @@ ip6 filter output log flags all
 								<entry>icmp_code</entry>
 							</row>
 							<row>
-								<entry>icmp6</entry>
+								<entry>icmpv6</entry>
 								<entry>ip6</entry>
 								<entry>icmpv6_code</entry>
 							</row>
-- 
1.8.3.1

